### ISSUE TITLE
VR mode support

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [google/model-viewer#4959]
There are a large number of XR headsets that either only support VR or support AR/MR, but only in native apps and not in the browser (WebXR).

E.g. until November 2022, Meta Quest didn't support AR in WebXR, only VR.

There are not a big number of users (e.g. in regions where Meta servers are blocked) and they can't update without a VPN.

Other users aren't updated for one reason or another intentionally. Not many of them either, but still.

AFAIK, currently Apple Vision Pro doesn't support AR in WebXR either.

And there are a bunch of wired VR headsets that connect to desktop computers, such as Oculus Rift, Valve Index, a bunch of Microsoft WMR (Windows Mixed Reality) devices, etc. that either don't transmit cameras stream to the desktop browser, or have no cameras at all and use base stations for tracking.

So how about VR mode support in `<model-viewer>`?

By VR mode I mean to use a special empty space / environment (e.g. a white scene). Something like https://youtu.be/ZoH9mUyyshA. And the motion controller models for dragging, scaling and rotating 3D models using motion controllers or hands, recently added by #4643.

This would significantly expand the list of devices that are supported by `<model-viewer>`.

**Comment by elalish** [comment]
Agreed, sounds good. We should probably build some kind of abstraction to hit testing so we can use the WebXR API in AR mode and normal scene raycasting in VR.
